kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -368,4 +523,1496 @@ def _document(cls):
+            - 'recalibrate' remedies a problem with 'resample'. 'resample'
+              calibrates all filters relative to longest input; when the shortest
+              input is very different in comparison, it makes most filters appear
+              lowpass-ish. In contrast, recalibration enables better exploitation
+              of fine structure over the smaller interval (which is the main
+              motivation behind wavelets, a "multi-scale zoom".)
+            - 'exclude' circumvents the problem by simply excluding wide filters.
+              'exclude' is simply a subset of 'resample', preserving all center
+              frequencies and widths - a 3D/4D coefficient packing will zero-pad
+              to compensate (see `help(kymatio.toolkit.pack_coeffs_jtfs)`).
+
+        Note: `sampling_phi_fr = 'exclude'` will re-set to `'resample'`, as
+        `'exclude'` isn't a valid option (there must exist a lowpass for every
+        fr input length).
+
+      max_pad_factor_fr : int / None (default) / list[int], optional
+        `max_pad_factor` for frequential axis in frequential scattering.
+
+            - None: unrestricted; will pad as much as needed.
+            - list[int]: controls max padding for each `N_fr_scales`
+              separately, in reverse order (max to min).
+                - Values must non-increasing (e.g. not `[2, 0, 1, ...]`)
+                - If the list is insufficiently long (less than number of scales),
+                  will extend list with the last provided value
+                  (e.g. `[1, 2] -> [1, 2, 2, 2]`).
+                - Indexed by `scale_diff == N_fr_scales_max - N_fr_scales`
+            - int: will convert to list[int] of same value.
+
+        Specified values aren't guaranteed to be realized. They override some
+        padding values, but are overridden by others.
+
+        Overrides:
+            - Padding that lessens boundary effects and wavelet distortion
+              (`min_to_pad`).
+
+        Overridden by:
+            - `J_pad_frs_min_limit_due_to_phi`
+            - `J_pad_frs_min_limit_due_to_psi`
+            - Will not allow any `J_pad_fr > J_pad_frs_max_init`
+            - With `sampling_psi_fr = 'resample'`, will not allow `J_pad_fr`
+              that yields a pure sinusoid wavelet (raises `ValueError` in
+              `filter_bank.get_normalizing_factor`).
+
+    pad_mode_fr : str (default 'conj-reflect-zero') / function
+        Name of frequential padding mode to use, one of: 'zero',
+        'conj-reflect-zero'.
+        Or, function with signature `pad_fn_fr(x, pad_fr, scf, B)`;
+        see `_right_pad` in
+        `kymatio.scattering1d.core.timefrequency_scattering1d`.
+
+        If using `pad_mode = 'reflect'` and `average = True`, reflected portions
+        will be automatically conjugated before frequential scattering to avoid
+        spin cancellation. For same reason there isn't `pad_mode_fr = 'reflect'`.
+
+    analytic : bool (default True)
```

Comment:
`analytic==True` ? Time scattering doesn't do this.
I'd rather play with the xi_max and the r_psi to adjust this.
Can we revert to `analytic==False`?